src/snowflake/connector/ocsp_snowflake.py

```diff
@@ -717,43 +703,27 @@ def is_cache_fresh(current_time, ts):
         return current_time - OCSPCache.CACHE_EXPIRATION <= ts
 
     @staticmethod
-    def find_cache(ocsp, cert_id, subject):
+    def find_cache(ocsp, cert_id, subject, **kwargs):
         subject_name = ocsp.subject_name(subject) if subject else None
         current_time = int(time.time())
-        hkey = ocsp.decode_cert_id_key(cert_id)
-        if hkey in OCSPCache.CACHE:
-            ts, cache = OCSPCache.CACHE[hkey]
+        cache_key = kwargs.get("cache_key", ocsp.decode_cert_id_key(cert_id))
```

Comment:
Please type-hint this line.
Maybe?
```suggestion
        cache_key: tuple[bytes, bytes, bytes] = kwargs.get("cache_key", ocsp.decode_cert_id_key(cert_id))
```
But that'd mean that my comment about `OCSP_RESPONSE_VALIDATION_CACHE`'s type-hint was correct

Reply:
you're correct, updated